assets/js/pages/ActivityLogPage/ActivityLogPage.test.jsx

```diff
@@ -15,7 +15,8 @@ const axiosMock = new MockAdapter(networkClient);
 describe('ActivityLogPage', () => {
```

Comment:
Forgot: we might want to add some tests making sure that given a state with some activity log users entries, those entries are in the filter options.

Reply:
At the page level, in order to see the user options inside the component, it requires user interaction (a click) and is therefore a better suited for an end-to-end test, in my understanding.

Reply:
A jest test here where we can simulate a user clicking the filter and asserting that expected options are there would be sufficient. There are plenty of examples on how to achieve that in the codebase.

Further scenarios would indeed be better suited for e2e tests but it might be costly implementing and the value returned minimal.
 
Your choice: a simple jest test here or a cypress e2e test.